src/shared/components/msktable/MSKTable.tsx

```diff
@@ -436,27 +437,37 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
         />
     }
 
-    render() {
+    buildTextBetweenTheButtons(): string {
```

Comment:
@adamabeshouse also moved `@computed get colVisProp(): IColumnVisibilityDef[]` to the store